.github/actions/runleaks/.github/workflows/scan_public.yml

```diff
@@ -0,0 +1,61 @@
+name: Scan Public Logs
+
+on: 
+  workflow_dispatch:
+    inputs:
+      repos:
+        description: 'Repos to scan:'
+        required: true
+        default: '[\"josiahsiegel\/runleaks\",\"josiahsiegel\/AzViz-action\"]'
```

Comment:
update repo

Reply:
All these issues are not documented and we are in the process of reaching out to Mr. Josiah Siegel and ask for his assistance in figuring out the different environment and secrets we need to setup in this project.